_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
@ashmaroli what's the value of delegated folders? Especially for a small boilerplate site? Anyone sophisticated enough to create a site with multiple javascript or css files will likely be sophisticated enough to not use the default theme. Put another way, if we do this, we have to assume every other theme does the same. Is that the precedence we want to set? 

@parkr can the new assets folder support support sub-folders?